beacon_node/lighthouse_network/src/config.rs

```diff
@@ -454,15 +445,15 @@ pub fn gossipsub_config(network_load: u8, fork_context: Arc<ForkContext>) -> Gos
     let gossip_message_id = move |message: &GossipsubMessage| {
         MessageId::from(
             &Sha256::digest(
-                prefix(MESSAGE_DOMAIN_VALID_SNAPPY, message, fork_context.clone()).as_slice(),
+                prefix(TSpec::default_spec().message_domain_valid_snappy, message, fork_context.clone()).as_slice(),
```

Comment:
same as above. We would need to thread a reference to the `ChainSpec` to the `gossipsub_config` function.